pyQuARC/schemas/check_messages.json

```diff
@@ -311,6 +311,22 @@
         },
         "remediation": "Recommend providing the instrument short name."
     },
+    "standard_product_check": {
+        "failure": "The Standard Product is missing.",
+        "help": {
+            "message": "",
+            "url": ""
+        },
+        "remediation": "Recommend indicating whether this is a StandardProduct. For information please see: https://wiki.earthdata.nasa.gov/display/CMR/StandardProduct"
+    },
+    "dif_standard_product_check": {
+        "failure": "The Standard Product is missing.",
+        "help": {
+            "message": "",
+            "url": ""
```

Comment:
Let's add "https://wiki.earthdata.nasa.gov/display/CMR/StandardProduct" in the space for url on line 326.